ci-operator/step-registry/cucushift/upgrade/toimage/cucushift-upgrade-toimage-ref.yaml

```diff
@@ -1,13 +1,10 @@
 ref:
   as: cucushift-upgrade-toimage
-  from_image:
-    namespace: ci
-    name: verification-tests
-    tag: latest
+  from: cloud-credential-operator
```

Comment:
Hmm, I am afraid this change would break some other upgrade scenarios, I can not remember exact reason why we used `verification-tests`, maybe related to rhel worker upgrade stuff, cc @shellyyang1989 to confirm.

Reply:
Using the image is for ccoctl tool:(

Reply:
We are using `verification-tests` is because we have asked productivity team to integrate `jq` and `ansible` in it. If we need `ccoctl`, we can add it to `verification-tests` as well.

Reply:
> If we need ccoctl, we can add it to verification-tests as well.

We had a simple discussion around it in the begging, considering it need maintain a dockerfile so drop it. But if we could not find better way, i think we should add it. @jianlinliu hdyt?
@shellyyang1989 could you share the link or jira issue for the requirement to add tool in base image `verification-tests`

Reply:
Here you go https://issues.redhat.com/browse/OCPQE-11640

Reply:
For the ccoctl, I think we can extract it from payload image just like what we did for oc client.

Reply:
Would u like a static tool pre-installed in the image or extract it dynamically in each upgrade? A little concern if it will bring one more unstable point to upgrade process.

Reply:
Let me update it to use dynamic extract way.

Reply:
updated